exist-core/src/main/java/org/exist/launcher/WindowsServiceManager.java

```diff
@@ -259,6 +259,25 @@ public void showNativeServiceManagementConsole() throws UnsupportedOperationExce
         }
     }
 
+    /**
+     * Try to find jvm.dll, which should either reside in `bin/client` or `bin/server` below
+     * JAVA_HOME. Autodetection does not seem to work with OpenJDK-based Java distributions.
+     *
+     * @return Path to jvm.dll or empty Optional
+     */
+    private Optional<String> findJvm() {
+        final Path javaHome = Paths.get(System.getProperty("java.home")).toAbsolutePath();
+        Path jvm = javaHome.resolve("bin/client/jvm.dll");
```

Comment:
If this is windows, should Paths not be written `\` instead of `/`? Also better not to use the separator at all and instead use `.resolve("bin").resolve("client").resolve("jvm.dll")

Reply:
@adamretter yep that is better

Reply:
Ok, changed.

Reply:
Thanks